docs/First-Installation.md

```diff
@@ -91,11 +87,15 @@ See the following diagrams for reference:
 |:---:|
 |![ps3_controller](https://cloud.githubusercontent.com/assets/10035308/22185413/f10f27de-e099-11e6-97a4-ecbbc82c9e46.png)|
 
+More info at [RetroArch Controller Configuration](RetroArch-Configuration)
+
 ### Hotkey
 
-The Hotkey button enables you to press it in combination with another button to access functions such as saving, loading, and exiting in emulators. It is suggested to use the **Select** button as the hotkey. The following chart shows the default hotkey combinations. For example, if you chose Select as your Hotkey, that means you hold down Select while pressing the other button to execute the command. 
+The Hotkey Enable button enables you to press it in combination with another button to access functions such as saving states, loading states, opening the RetroArch GUI (RGUI), and exiting back to EmulationStation. It is recommended to define an unused button or key as your Hotkey Enable button. For example, the *Home* button on some controllers such as the center "X" on Xbox controllers. If your controller doesn't have an unused button, the *Select* button is a good alternative.
```

Comment:
Please keep **bold** to designate buttons ("**Select**"). I believe the general use across the docs, although in truth it's probably not particularly consistent. Still, if we're going change it, it should be discussed and done wholesale in another PR.

Reply:
Select *and* Home, since Home is also a button? Or does this mean I should wait until that discussion has been done?

Reply:
I'm indifferent about 'Home' since it's a hardware designation and not a software designation within RetroPie so either way is fine.